src/fondant/core/schemas/common.json

```diff
@@ -57,6 +57,11 @@
     "fields": {
       "type": "object",
       "minProperties": 1,
+      "properties": {
```

Comment:
Can you explain how you want to implement this now? It's not clear which the final design is to me. As mentioned above, it's fine for me to leave this out for now and add it as a separate PR.